internal/dynamo/client.go

```diff
@@ -375,6 +399,32 @@ func (c *Client) DeleteOne(ctx context.Context, pk PK, sk SK) error {
 	return err
 }
 
+func (c *Client) DeleteManyByUID(ctx context.Context, keys []Keys, uid string) error {
```

Comment:
I don't think there is any point having `uid` again, we should be able to trust the previous query

Reply:
so could use `DeleteKeys`

Reply:
I know the likelihood of getting exactly the same timestamp + action type is low but as the keys aren't unique I wanted a failsafe there to be sure we weren't deleting unintended events. If you're comfortable with that risk feel free to drop this while I'm out and just rely on the keys 👍

Reply:
every (PK, SK) is unique, so it won't be an issue. I have spotted `Put` doesn't use `Create`, so it would currently overwrite an existing scheduled event, I'm going to change that